lib/ukofw/Makefile.uk

```diff
@@ -0,0 +1,8 @@
+$(eval $(call addplatlib_s,kvm,libukofw,$(CONFIG_LIBUKOFW)))
+
+LIBUKOFW_CINCLUDES-y         += -I$(LIBUKOFW_BASE)/include
```

Comment:
These need to be conditional to `CONFIG_LIBUKOFW`, otherwise they are included even when the library is not enabled, eg:
```
LIBUKOFW_CINCLUDES-$(CONFIG_LIBUKOFW)
```
Similarly for the rest.